setup.py

```diff
@@ -25,8 +25,9 @@
 version = "0.2.0"
 release_status = "Development Status :: 4 - Beta"
 dependencies = [
-    "google-api-core[grpc] >= 1.14.0, < 2.0.0dev",
-    'enum34; python_version < "3.4"',
+    "google-api-core[grpc] >= 1.22.0, < 2.0.0dev",
+    "proto-plus >= 0.4.0",
```

Comment:
```suggestion
    "proto-plus >= 1.4.0",
```

This seems to have been bumped a few times recently in the templates. https://github.com/googleapis/gapic-generator-python/blob/master/gapic/templates/setup.py.j2#L21